### PR TITLE
add chatBot

### PR DESCRIPTION
**Pull request by Lizaveta0302**
The server-client chatBot
The client connects to the server and server sends a welcome msg, that is displayed in the server's and client console and then the server reacts to client messages and send a response.

### CODE REVIEW

File: src/main/chatBot/BotLogic.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+        switch (state) {
+            case START: {
+                output = "Hi, I am a super Bot! I am a psychologist! Do you want to tell me about your problems?(yes/no)";
+                state = State.PROCEED;
+                break;
+            }
+            case PROCEED: {
+                if (input.equalsIgnoreCase("yes")) {
+                    output = feels[0].concat(questions[0]).concat(questions[1]);
```
Comment (Savotii) [inline src/main/chatBot/BotLogic.java]: badPractice, wrap this part in a particular method.

Comment (Savotii) [reply src/main/chatBot/BotLogic.java]: > 
I mean whole case Proceed
>

>

File: src/main/chatBot/Client.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            communicateWithServer(in, out, inputLine);
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private void communicateWithServer(PrintWriter in, BufferedReader out, BufferedReader inputLine) throws IOException {
+        String answerFromServer, input;
+        while (true) {
```
Comment (Savotii) [inline src/main/chatBot/Client.java:34]: Does this method finish?

File: src/main/chatBot/Server.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            e.printStackTrace();
+        }
+    }
+
+    private void startConversation(PrintWriter in, BufferedReader out) throws IOException {
+        String input, answer;
+        BotLogic botLogic = new BotLogic();
+        answer = botLogic.process(null);
+        in.println(answer);
+        while (true) {
```
Comment (Savotii) [inline src/main/chatBot/Server.java:36]: Does this method finish?

File: src/main/chatBot/BotLogic.java (hunk 1 above)
Comment (Lizaveta0302) [reply src/main/chatBot/BotLogic.java]: understood, I will do it now

Comment (Lizaveta0302) [reply src/main/chatBot/BotLogic.java]: fixed it

Comment (Lizaveta0302) [reply src/main/chatBot/BotLogic.java]: can I merge it?

**Pull request merged**